src/main/java/com/provectus/collections/homeTasks/NoMoreDuplicates.java

```diff
@@ -0,0 +1,64 @@
+package com.provectus.collections.homeTasks;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+/**
+ * @author Victor Yemelianenko vemelianenko@model.com
+ */
+
+public class NoMoreDuplicates {
+    public static void main(String[] args) {
+        Random random = new Random();
+
+        List<Integer> randomListWithDuplicate = new ArrayList<>();
+
+        for (int i = 0; i < 20; i++) {
+            randomListWithDuplicate.add(random.nextInt(10));
+        }
+        System.out.println(randomListWithDuplicate);
+        long start0 = System.currentTimeMillis();
+        for (int i = 0; i < randomListWithDuplicate.size(); i++) {
```

Comment:
we can simplify this, use only one loop for the initial list, and check one condition: if new list does not contain element from the initial list, then add it to the new list